packages/atomic/src/components/search/tabs/atomic-tab-section/atomic-tab-section.tsx

```diff
@@ -0,0 +1,185 @@
+    const tabs = [...this.host.querySelectorAll('atomic-tab')];
+    this.tabs = tabs;
+
+    if (tabs.length === 0) {
+      this.error = new Error(
+        'The "atomic-tab-section" element requires at least one "atomic-tab" child.'
+      );
+      return;
+    }
+  }
+
+  setInitialTab() {
+    const initialTab =
+      this.tabs.find((tab) => tab.name === this.defaultActiveTab) ||
+      this.tabs[0];
+    const activeTab = getActiveTab(this.bindings.engine.state)?.tab;
+
+    if (initialTab && !activeTab) {
+      this.bindings.engine.dispatch(
+        loadTabSetActions(this.bindings.engine).updateActiveTab(initialTab.name)
+      );
+      this.tabInit.emit();
+    }
+    this.updateActiveTab();
+  }
+
+  buildDropdown() {
+    return this.tabs.map((tab) => (
+      <li>
+        <Button
+          style="text-transparent"
+          title={tab.name}
+          class="w-full px-6 pb-1 text-xl text-neutral-dark dropdown-option"
+          text={tab.label}
+          onClick={() => {
+            tab.select();
+          }}
+        />
+      </li>
+    ));
+  }
+
+  updateActiveTab() {
+    const dropdownButton =
+      this.host.shadowRoot?.querySelector('.dropdown-button');
+    const dropdownOptions = [
+      ...(this.host.shadowRoot?.querySelectorAll('.dropdown-option') || []),
+    ];
+    const activeTabName = getActiveTab(this.bindings.engine.state)?.tab;
+
+    this.tabs.forEach((tab) => {
+      tab.isActive = tab.name === activeTabName;
+
+      if (tab.isActive) {
+        const span = dropdownButton?.getElementsByTagName('span')[0];
+        if (span) {
+          span.textContent = tab.label;
+        }
+      }
+    });
+
+    dropdownOptions.forEach((option) => {
+      if ((option as HTMLButtonElement).title === activeTabName) {
+        option.classList.add('font-bold');
+      } else {
+        option.classList.remove('font-bold');
+      }
+    });
+  }
+
+  clearState() {
+    const breadcrumbActions = loadBreadcrumbActions(this.bindings.engine);
+    const queryActions = loadQueryActions(this.bindings.engine);
+    const searchActions = loadSearchActions(this.bindings.engine);
+    const searchAnalyticsActions = loadSearchAnalyticsActions(
+      this.bindings.engine
+    );
+
+    this.bindings.engine.dispatch(breadcrumbActions.deselectAllBreadcrumbs());
+    this.bindings.engine.dispatch(
+      breadcrumbActions.deselectAllNonBreadcrumbs()
+    );
+    this.bindings.engine.dispatch(queryActions.updateQuery({q: ''}));
+    this.bindings.engine.dispatch(
+      searchActions.executeSearch(searchAnalyticsActions.logInterfaceChange())
+    );
+  }
+
+  public render() {
+    return (
+      <div class="mb-4 overflow-visible ">
+        <div class="flex flex-row tabs-container">
+          <div class="flex flex-row w-full mb-2 border-b tabs-area ">
+            <slot></slot>
+          </div>
+          <div class="relative flex-row hidden mb-2 border-b cursor-pointer group dropdown-area ">
+            <Button
```

Comment:
There's quite a few accessibility problem with trying to simulate what is essentially a `<select>` with buttons/dropdown and the likes with JS code.

Then you can add in mobile support, which is much better when you use the correct semantic element, versus a cramped up dropdown.

Reply:
I checked what would need to be done to make it more accessible. It could be done relatively easily, but it would require adding the aria-haspopup attribute to our button component. However, I decided to just go with a select+option as you suggested :)